docs/multiplatform.md

```diff
@@ -64,17 +64,22 @@ Add the compiler dependencies in a top level `dependencies` block
 ```kotlin
 dependencies {
     // 1. Configure code generation into the common source set
-    kspCommonMainMetadata(libs.kotlinInject)
+    kspCommonMainMetadata(libs.kotlin.inject.compiler)
 
     // 2. Configure code generation into each KMP target source set
-    kspAndroid("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
-    kspIosX64("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
-    kspIosArm64("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
-    kspIosSimulatorArm64("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
+    listOf(
```

Comment:
I think it's simpler to list the standard way of declaring the dependency, and let the reader adopt their preferred Kotlin sugar on top of that.